source/_components/konnected.markdown

```diff
@@ -116,6 +116,23 @@ devices:
         repeat:
           description: Number of times to repeat a momentary pulse. Set to `-1` to make an infinite repeat. This is useful as an alarm or warning when used with a piezo buzzer.
           required: false
+    host: 
+      type: string
+      required: false
+      description: Optionally specify the Konnected device's IP address or hostname to set up without discovery.
+    port:
+      type: integer
+      required: false
+      description: Optionally specify the port number for the Konnected API on the device. Note that the port is different on every device. See help.konnected.io to learn how to determine the port number.
+    discovery:
+      type: boolean
+      required: false
+      description: Enable or disable discovery for this device. When true, the device will respond to discovery requests on your network. When false, the device will not respond to discovery requests, so it's important that you set reserved IP for the device and configure the _host_ and _port_ here.
```

Comment:
Use code quotes (that backwards single quote thing) when talking about boolean values in the description:

- true  --> \`true\`
- false --> \`false\`